locations/spiders/salad_story_pl.py

```diff
@@ -0,0 +1,45 @@
+from scrapy import Request, Spider
+
+from locations.categories import Categories
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_FULL, OpeningHours
+
+
+class SaladStoryPLSpider(Spider):
+    name = "salad_story_pl"
+    item_attributes = {
+        "brand": "Salad Story",
+        "extras": {"amenity": Categories.FAST_FOOD.value, "cuisine": "salad", "diet:vegetarian": "yes"},
```

Comment:
`Categories.FAST_FOOD.value` is a dict itself, so it needs to look like this:

```suggestion
        "extras": {
                **Categories.FAST_FOOD.value,
                "cuisine": "salad",
                "diet:vegetarian": "yes",
        },
```

Reply:
Thanks, I had some concerns about that part.